padd.sh

```diff
@@ -500,13 +502,19 @@ GetNetworkInformation() {
         # No IPv6 address available
         pi_ip6_addr="N/A"
         ipv6_check_box=${check_box_bad}
+        ipv6_status="Disabled"
+        ipv6_heatmap=${red_text}
```

Comment:
Do you consider IPv6 "disabled" as an error (red)?

Maybe we could use another color, just like we did for https://github.com/pi-hole/PADD/issues/64

Do you think this should be considered an alert (yellow, like `dhcp_range_heatmap`) or just disabled?

Reply:
Good question. We have `ipv6_check_box` set to 'bad' for 3 years now when no IPv6 address is found and no one complaint so far.
I can see arguments for both: it's 2025 and each device should have an IPv6 by now (can be even link-local). On the other hand, some users might disable IPv6 completely on purpose....

Reply:
Personally, I don't think the absence of IPv6 is a failure. Sometimes it is an option.
Also, IPv6 is usually disabled in docker containers.

In my personal opinion we can use:
- red, if it's a failure and it needs to be corrected;
- yellow, if the item needs attention or it is an unexpected state;
- blue, if it's something we just need to inform, using a different color just to distinguish between (on/off, present/absent), but it's not an issue.

Reply:
Ok, convinced. I pushed an update.